distributed/worker.py

```diff
@@ -465,6 +465,12 @@ def upload_file(self, comm, filename=None, data=None, load=True):
                         reload(import_module(pkg.project_name))
                     if not pkgs:
                         logger.warning("Found no packages in egg file")
+                if ext == '.zip':
+                    logger.info("Reload module %s from .zip file", name)
+                    if out_filename not in sys.path:
+                        sys.path.insert(0, out_filename)
+                    name = name.split('-')[0]
```

Comment:
I kept this line from ('.py', '.pyc') case
```
                if ext in ('.py', '.pyc'):
                    logger.info("Reload module %s from .py file", name)
                    name = name.split('-')[0]
                    reload(import_module(name))
```
Actually I don't know what it is for, so i can remove it in the zip case